project_transport_only/project.json5

```diff
@@ -18,7 +18,7 @@
                         subsector: {
                             base: [
                                 'Personal_LDV+Compact+BEV_200', 
-								'Personal_LDV+Compact+BEV_300',
+                'Personal_LDV+Compact+BEV_300',
```

Comment:
These indents still look off in GitHub, just with too little indent instead of too much. Still need to bulk convert some tabs to spaces?

Reply:
There is a lot of formatting incorrect in this file. I'll avoid introducing more possible merge conflicts with Arthur. Perhaps I shouldn't have changed the tabs.